packages/scaffold/src/client.ts

```diff
@@ -196,6 +228,9 @@ export class Web3ModalScaffold {
 
     ConnectionController.setClient(options.connectionControllerClient)
 
+    if (options.siweControllerClient) {
```

Comment:
if SIWEControler can be undefined (optional) you can remove an if statement

Reply:
`SIWEController.setSIWEClient(client)` expects `client` to be defined.